test/hexpm/repository/release_test.exs

```diff
@@ -483,6 +483,31 @@ defmodule Hexpm.Repository.ReleaseTest do
            ] = release.requirements
   end
 
+  test "update release fails with invalid version", %{publisher: publisher, packages: [_, package2, package3]} do
+    Release.build(package3, publisher, rel_meta(%{version: "0.0.1", app: package3.name}), "")
+    |> Hexpm.Repo.insert!()
+
+    reqs = [%{name: package3.name, app: package3.name, requirement: "~> 0.0.1", optional: false}]
+
+    release =
+      Release.build(
+        package2,
+        publisher,
+        rel_meta(%{version: "0.0.1", app: package2.name, requirements: reqs}),
+        ""
+      )
+      |> Hexpm.Repo.insert!()
+
+    params =
+      params(%{
+        app: package2.name,
+        version: "1.0",
+      })
+
+      assert %Ecto.Changeset{} = error =  Release.update(%{release | package: package2}, publisher, params, "") 
```

Comment:
This is causing the build to fail. Yet if I change this to match on `"is invalid Simver"` it fails locally.